### PR TITLE
Increase test stability and cleanup some logging/debug

### PR DESCRIPTION
**Pull request by grafnu**

### CODE REVIEW

File: validator/src/main/java/com/google/daq/mqtt/registrar/LocalDevice.java [hunk 1]
```diff
@@ -309,7 +310,7 @@ private Metadata readMetadataWithValidation(boolean validate) {
     final JsonNode instance;
     try (InputStream targetStream = new FileInputStream(metadataFile)) {
       instance = OBJECT_MAPPER.readTree(targetStream);
-      baseVersion = instance.get("version");
+      baseVersion = instance.get(VERSION_PROPERTY_KEY);
       new MessageUpgrader("metadata", instance).upgrade();
```
Comment (johnrandolph) [inline validator/src/main/java/com/google/daq/mqtt/registrar/LocalDevice.java:314]: Should this be MessageUpgrader.METADATA_SCHEMA

File: validator/src/main/java/com/google/daq/mqtt/sequencer/SequenceBase.java [hunk 2]
```diff
@@ -392,6 +393,9 @@ private Config readGeneratedConfig() {
    */
   @Before
   public void setUp() {
+    // Old messages can sometimes take a while to clear out, so need some delay for stability.
+    safeSleep(CLEAN_START_DELAY_MS);
```
Comment (johnrandolph) [inline validator/src/main/java/com/google/daq/mqtt/sequencer/SequenceBase.java:397]: Can we add a TODO to minimize this time or change the strategy?

**Review by johnrandolph — APPROVED** [review]

File: validator/src/main/java/com/google/daq/mqtt/sequencer/SequenceBase.java (hunk 2 above)
Comment (grafnu) [reply validator/src/main/java/com/google/daq/mqtt/sequencer/SequenceBase.java:397]: Done.

File: validator/src/main/java/com/google/daq/mqtt/registrar/LocalDevice.java (hunk 1 above)
Comment (grafnu) [reply validator/src/main/java/com/google/daq/mqtt/registrar/LocalDevice.java]: Done.

**Pull request merged**